proposals/handler.go

```diff
@@ -45,6 +45,7 @@ var (
 	errKnownBallot           = errors.New("known ballot")
 	errInvalidVote           = errors.New("invalid layer/height in the vote")
 	errMaliciousBallot       = errors.New("malicious ballot")
+	errWrongSmesherID        = errors.New("ballot atx from a different smesher")
```

Comment:
i think it is not really necessary to add such errors if this is only for information and testing. in both cases string-based error seems to be sufficient

Reply:
noted.